### PR TITLE
Non functional card selector for home page

### PR DESCRIPTION
**Pull request by krischarbonneau**
### Description

This PR adds a card selector to the home page, with default values selected as per Figma designs. When a user selects (or unselects) a card from the list, the value is either added to or removed from the `cardList` array (which will later be used to update the room upon creation). I've chosen to stay away from `useState` for updating the card list since the value was only getting added after state had been updated, which is not ideal. Instead I used `useRef` which updates the array in real time, but since this is not communicated to the browser, there is no re-render and as such we run into problems with rendering the `selected` styles on the cards when they are clicked. Adding the `forceUpdate` reducer and calling it whenever a card is clicked fixes this, but I am open to suggestions if this isn't ideal.

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Enter a name and unselect all the cards
4. Click create room, you should be shown an error asking you to select at least one card
5. Select a card and click create room again, you should be redirected to a room (the cards you selected won't be the ones shown in the room as this functionality isn't in this PR)

### Additional Notes

I've also moved the array of card SVGs into a separate file that can be imported instead of doubling up on code on the home and room pages. I'll make use of this when I have the room communicate with the server to display the selected cards.

### CODE REVIEW

File: utils/cards.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+export const cards = [
```
Comment (DaceyTom2) [inline utils/cards.js:1]: Ah yes I'm so glad this finally happened! This is 100% better outside of the pages/components

File: pages/home.js [hunk 2]
```diff
@@ -23,6 +25,10 @@ export default function Home(props) {
   const [createRoomError, setCreateRoomError] = useState(undefined)
   const [joinRoomError, setJoinRoomError] = useState(undefined)
 
+  //useRef doesn't notify the browser when mutating the current ref, so forceUpdate allows us to trigger a re-render instead of using useState and having a delay
+  const [, forceUpdate] = useReducer((x) => x + 1, 0)
+  const cardList = useRef([0, 1, 2, 3, 5, 8, 13, 20, 1001, 1002])
```
Comment (DaceyTom2) [inline pages/home.js:30]: I'm not convinced this a good strategy. Though I like the idea of introducing reducers but they would be better fit for the users/room state on the room page

**Review by DaceyTom2 — APPROVED** [review]

**Comment by shewood** [comment]
One thing that I noticed is when trying to remove a card from the deck, the card remains shown in a selected state until you click in another place. I was expecting the card to show as deselected when doing so.

**Review by P0NDER0SA — APPROVED** [review]
Looks good.  let's get it merged

**Pull request merged**